tools/firmware-utils/src/mkrtn56uimg.c

```diff
@@ -252,12 +253,12 @@ int
 main(int argc, char **argv)
 {
 	int 		opt;
-	char 		*filename, *progname;
+	char 		*filename, *progname, *productid = "RT-N56U";
```

Comment:
I'd prefer to remove the (now arbitrary) default value here entirely and add it as parameter to the existing device(s) in image/*.mk as you do it for your new device.

Reply:
Done